webview/src/utilities/get-workflow-tasks.ts

```diff
@@ -69,16 +70,20 @@ function categorizeEvent(eventType: EventType | undefined): Category {
     case EventType.EVENT_TYPE_EXTERNAL_WORKFLOW_EXECUTION_SIGNALED:
     case EventType.EVENT_TYPE_SIGNAL_EXTERNAL_WORKFLOW_EXECUTION_FAILED:
     case EventType.EVENT_TYPE_REQUEST_CANCEL_EXTERNAL_WORKFLOW_EXECUTION_FAILED:
+    case EventType.EVENT_TYPE_WORKFLOW_UPDATE_REQUESTED:
+    case EventType.EVENT_TYPE_WORKFLOW_PROPERTIES_MODIFIED:
```

Comment:
I mislead you here, `EVENT_TYPE_WORKFLOW_PROPERTIES_MODIFIED` is a command not an event.